### ISSUE TITLE
Bluetooth support

### ISSUE THREAD

**Issue by ballo** [soduto/Soduto#32]
I read online that KDE Connect supports bluetooth, but Soduto doesn't appear to. Would it be possible to add this feature? I'm at my wit's end trying to copy a link to my Mac from my Pixel 7.

**Comment by ballo** [comment]
I'm going to leave a work-around here in case anyone else is having similar difficulty. 

Android no longer supports the bluetooth file transfer standard and has their own BS now. The work-around is to download the app "Total Commander" which has built-in support for it.

**Comment by sannidhyaroy** [comment]
> I'm at my wit's end trying to copy a link to my Mac from my Pixel 7. 

Soduto already syncs your clipboard, so you just copy the link on your Pixel 7 and paste on your mac. Alternatively, you can open the link in a browser on your Pixel and then tap on share and select KDE/Zorin Connect and choose which device to send the link to...

**Comment by ballo** [comment]
> > I'm at my wit's end trying to copy a link to my Mac from my Pixel 7.
> 
> Soduto already syncs your clipboard, so you just copy the link on your Pixel 7 and paste on your mac. Alternatively, you can open the link in a browser on your Pixel and then tap on share and select KDE/Zorin Connect and choose which device to send the link to...

I don't think you read the title? If soduto was connecting, this wouldn't be an issue, howerver soduto requires that all devices be on the same wifi LAN and that the router allows peers to broadcast and communicate. This only works for me when I use my hotspot. Not all of us are using this with a home network or whatever setup you have.
